src/obfuscation/hidden.c

```diff
@@ -0,0 +1,123 @@
+#include "hidden.h"
+#include "rmalloc.h"
+#include "util/minmax.h"
+#include "redis_index.h"
+#include "query_node.h"
+#include "reply_macros.h"
+
+typedef struct {
+  const char *user;
+  uint64_t length;
```

Comment:
Why not use `size_t` instead of `uint64_t`?
(`size_t` is portable, sized accordingly, and is also used by other library functions)